### DISCUSSION TITLE
Can I use ReactFlow directly in .html file?

### DISCUSSION THREAD

**Discussion by kuldeepReact123** [xyflow/xyflow#3096] [Q&A] [answered]
I am using ReactFlow in a component of my react app and the same I want to show in a pdf report(when generating from UI) as well . For a pdf I am creating a .html file on my machine locally by setting up static data and sending to BE and then they are replacing static data with dynamic values. In short, from UI prospective all I have to show the ReactFlow diagram/topology in a pdf report being generated from the .html file. So is there any way to use ReactFlow in html file? 

Thanks in advance for any help.

**Comment by chrtze** [comment, marked as answer]
Hey, you can generate an image and include it in the pdf: https://reactflow.dev/docs/examples/misc/download-image/
